### ISSUE TITLE
Can/should we add something to connect related topics?

### ISSUE THREAD

**Issue by caro401** [codexfelis/oratio#11]
There's a few sets of related topics that came up in article ideas - AI, git/hub, setting up your coding environment. It might be nice to link these together into some sort of more structured read this, then this, then this and you'll be able to do X and Y?

Sort of like a curriculum but less formal. The word that came to mind was "learning path" but I'm not sure I like it. Just as a way to provide an entrypoint to a topic, rather than make people go jumping around or searching if they didn't arrive here in search of something very specific?

Example (with a bunch of imagined articles) for a path to being confident interacting with other people's code on github:
- Intro to the path: you know nothing about version control, but are either curious or need to use it with a team or an open source project for work. We assume you know nothing at the start. At the end, you'll be able to use git for your own stuff, and with a team via github. Go read these things in this order
  - what's version control and why do i care
  - what's git
  - what's github
  - how to git alone
  - how to write a good issue?
  - how to PR
- Yay, now you know how to git. Go make a PR somewhere (maybe here!). Here's some additional resources from other places if you want to learn more (annotated list)

@danwaterfield do you think that would be a useful thing? Where would they live? Probably a first section in the sidebar? What should we call them?

Sorry for word vomit, insufficiently caffeinated but idea needed to come out before I forget it.

**Comment by caro401** [comment]
MDN's new [curriculum](https://developer.mozilla.org/en-US/blog/mdn-curriculum-launch/) feature is almost exactly what I had in mind when I wrote this issue

**Comment by danwaterfield** [comment]
Yes, links to what I was thinking of re: ai safety, in that people talk
about xyz from a uniquely technical perspective rather than holistically.
